### PR TITLE
Remove dependence on lodash

### PR DESCRIPTION
**Pull request by daytonn**

### CODE REVIEW

File: bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js [hunk 1]
```diff
@@ -25,19 +23,21 @@ module.exports = (function () { // eslint-disable-line no-unused-vars
     this.defaultText = element.innerHTML;
     this.loadingText = 'Loading...';
 
-    defaults(options, {
+    options = Object.assign({
```
Comment (kand) [inline bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js]: I don't believe `Object.assign` is available in IE 11.

**Comment by MelizzaP** [comment]
👍

File: bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js [hunk 2]
```diff
@@ -1,9 +1,7 @@
 module.exports = (function () { // eslint-disable-line no-unused-vars
-  var _ = require('lodash');
-  var assign = _.assign;
-  var defaults = _.defaults;
-  var isUndefined = _.isUndefined;
-  var trim = _.trim;
+  function isUndefined (suspect) {
+    return suspect === undefined; // eslint-disable-line no-undefined
```
Comment (kand) [inline bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js]: I think it's better form to do `typeof(suspect) === 'undefined'`, avoids the eslint error probably as well.

Comment (daytonn) [inline bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js]: I actually just copied the implementation from lodash, I don't see any benefit in using typeof here.

Comment (daytonn) [inline bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js]: https://github.com/lodash/lodash/blob/4.15.0/lodash.js#L11880-L11882

Comment (kand) [inline bulbs/special_coverage/static/special_coverage/js/special-coverage-loader.js]: We'd have a real broken site if it were the case, but it avoids the edge case where `undefined` is reassigned. No reason to write more bulletproof code here when it's only a few more characters.

Lodash actually protects for this edge case in a different way here: https://github.com/lodash/lodash/blob/4.15.0/lodash.js#L11-L12

**Pull request merged**